agent/resources/resources_linux.go

```diff
@@ -55,12 +70,18 @@ func (c *cgroupWrapper) Cleanup(task *api.Task) error {
 	return c.cleanupCgroup(task)
 }
 
-// cgroupInit is used to create the root '/ecs/ cgroup
+func (c *cgroupWrapper) ApplyConfigDependencies(cfg *config.Config) {
+	c.cgroupPath = cfg.CgroupPath
+}
+
+// cgroupInit is used to create the root '/ecs/ cgroup and enable
+// memory.use_hierarchy at the '/ecs/' level
```

Comment:
Can you edit this comment to reflect the new changes?

Reply:
good catch.